spec/cli_spec.rb

```diff
@@ -680,6 +680,27 @@
     end
   end
 
+  context "#set_date_time" do
+    let(:datetime) { double("date time") }
```

Comment:
I feel like it would be much better do use a real instance here.
This would allow you to make assertions on a real object rather than stubbing setters and getters for attributes.

```ruby
let(:datetime) { ManageIQ::ApplianceConsole::DateTimeConfiguration.new }

before do
  expect(ManageIQ::ApplianceConsole::DateTimeConfiguration).to receive(:new).and_return(datetime)
end
```